src/clj/game/core/diffs.clj

```diff
@@ -352,7 +353,9 @@
 (defn run-summary
   [state]
   (when-let [run (:run @state)]
-    (select-non-nil-keys run run-keys)))
+    (-> run
+        (update :current-ice #(some? (get-card state %)))
```

Comment:
I think we should either have `:current-ice` actually hold the current ice, or we should name it something else. Better to keep consistency between front and back end. It's extra work to remember that `:current-ice` is a card object in the engine but a boolean in the cljs.

Reply:
Yeah, I wasn't sure what the best way to handle it is. I really just need to know if the approached ice is still there. I think the most intuitive thing is to just put `:current-ice` through `card-summary` but it's more than I really need and we have to make sure hidden information is stripped so the Runner can't see what an unrezzed approached ice is. 

I think our options are:
1. Rename this to like `:current-ice-exists?` (or some better name I cannot think of)
2. Move the `run-summary` out of `strip-state` and into `state-summary` where we can pass it the side and properly strip the card

I'm leaning towards 2, but I feel like moving `run-summary` to `state-summary` goes against some of the work you did in refactoring this file.

Reply:
I think I'd prefer 1 because there's no reason to include the `:current-ice` when it can be/is derived by the client. I think I added `:current-ice` in the engine to handle when an ice is trashed but we still need to refer to it, but it's an ugly hack. Better to not send anything at all and rely on existing stripping to know if a given card is visible or not.